frontend/config-assistant/src/components/toolbar/clearSchema.ts

```diff
@@ -22,17 +22,10 @@ export function newEmptySchemafile(message: string | undefined = undefined): voi
       });
       clearSchema();
     },
-    reject: () => {
-      toastService.add({
-        severity: 'error',
-        summary: 'Rejected',
-        detail: 'Schema is kept',
-        life: 3000,
-      });
-    },
   });
 }
 export function clearSchema() {
+  newEmptySchemafile('Do you want to clear the Schema editor?');
```

Comment:
This will not work:

line 28 will trigger a "Do you want to clear schema?" dialog, which will then call the clearSchema function.

And line 30 will also clear the schema, independent of the user selection. 

So even if the user decides to not clear the schema, it will clear the schema.

Suggestion:

add a new function `openClearSchemaDialog()` to `clearSchema.ts`, which calls `newEmptySchemafile('Do you want to clear the Schema editor?');`

Then remove the newEmptySchemafile call from the clearSchema function.

Reply:
done